BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -809,7 +809,9 @@ Registry"), within the
    groups.  Curves of less than 224 bits MUST NOT be used. This recommendation is in-line with the latest
 revision of {{NIST.SP.800-56A}}. 
 
-When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}} for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}}[^live-ref-question] for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
```

Comment:
The "live-ref-question" thing doesn't render.

Reply:
oh right I didn't notice that [2fb0e66](https://github.com/yaronf/I-D/pull/428/commits/2fb0e662ef223b8201d7914aa2be7850b3b260a0) had removed the anchor.

will fix it (along with the other comment you have below)

Reply:
done in [f229a1e](https://github.com/yaronf/I-D/pull/428/commits/f229a1ee1ee411caf62e54526778bd1a2d1edac0)